src/mscorlib/Resources/Strings.resx

```diff
@@ -293,7 +352,7 @@
     <value>Property Get method was not found.</value>
   </data>
   <data name="Arg_GuidArrayCtor" xml:space="preserve">
-    <value>Byte array for GUID must be exactly {0} bytes long.</value>
+    <value>Constructor for Guid needs exactly {0} bytes.</value>
```

Comment:
It looks like when you edited this with Visual Studio, it made a few other changes automatically.  Might make sense to revert the changes to this file and then just use notepad to make this one line change.